libcnb-data/src/buildpackage.rs

```diff
@@ -0,0 +1,92 @@
+use crate::buildpackage::PlatformOs::Linux;
+use serde::{Deserialize, Serialize};
+
+fn platform_default() -> Platform {
+    Platform::default()
+}
+
+#[derive(Debug, Deserialize, Serialize, Clone)]
+#[serde(deny_unknown_fields)]
+pub struct Buildpackage {
+    pub buildpack: BuildpackageUri,
+    #[serde(default)]
+    pub dependencies: Vec<BuildpackageUri>,
+    #[serde(default = "platform_default")]
+    pub platform: Platform,
+}
+
+#[derive(Debug, Deserialize, Serialize, Eq, PartialEq, Clone)]
+#[serde(deny_unknown_fields)]
+pub struct BuildpackageUri {
+    pub uri: String,
```

Comment:
See comment above about the type in general. In addition, can we make this field a `Uri`? `libcnb-data` tries to parse into stricter types where we can and I think this is such a case. This also would make the upstream code that uses this field a little easier, because it will always be a valid URI.